packages/legend-application-query/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -497,6 +498,11 @@ export const QueryBuilderResultPanel = observer(
                 className="query-builder__result__execute-btn"
                 onClick={runQuery}
                 tabIndex={-1}
+                title={` ${
```

Comment:
--> `title={queryBuilderState.isValidQueryBuilderState() ? 'Query is not valid' : undefined}